iree/compiler/JitEval/BUILD

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 The IREE Authors
```

Comment:
I think JIT may be too overloaded a term to use here - maybe ConstEval or ConstExprEval or something that makes it clear what it's for and matching the naming used in the analysis

Reply:
Renamed to ConstEval and cleaned up naming a bit more.